EDI/Operation/01_ManageDefinitions.md

```diff
@@ -0,0 +1,249 @@
+
+
+#### Procedure
+
+*DataHub Exporter > Exports*
+
+![Export definitions](../../Assets/Screenshots/EDI/Operation/ExportDefinitions.png "[Export definitions]")
+
+1. Click the checkbox of the definition you want to preview. Alternatively, you can click the *Download link* of a definition in the list to output the preview directly.   
+    The editing toolbar is displayed. 
+    
+
+    ![Editing toolbar](../../Assets/Screenshots/EDI/Operation/ExportDefinitionToolbar.png "[Editing toolbar]")
+
+2. Click the [PREVIEW] button.   
+    The window for specifying the output is displayed. You can ignore the *Export all* toggle. It is relevant for mass transports only.
+
+    ![Preview](../../Assets/Screenshots/EDI/Operation/ExportDefinitionPreview.png "[Preview]")
+
+3. Click the *Entity type* drop-down list and select the data sources you currently use for this definition, for example the **Retail suite business document** data source for the header export definition or the **Retail suite business document position** data source for the line item export definition. 
+
+4. Enter **31** in the *Primary identifier* field.
+
+5. Click the [OPEN PREVIEW] button. 
+    The syntax of the message(s) is output in a new browser window. 
+
+6. Copy the output. If required, send it to your business partner for validation. 
+
+
+
+## Activate definition
+
+After you have configured the entire header and line item definitions including the file settings, activate the definition to be able to send EDIFACT messages to your business partner.
+
+#### Prerequisites
+
+- At least one connection has been created, see [Create connection](../Integration/01_ManageConnections.md#create-connection).
+- At least one definition setting has been created, see [Create basic definition settings](#create-basic-definition-settings).
+- You have defined the file settings for this definition, see [Manage header file settings](./02_ManageHeaderFileSett.md#manage-header-file-settings) and [Manage line item file settings](./04_ManageLineItemsFileStt.md#manage-line-item-file-settings).
+- You have defined the data sources for this definition, see [Assign data source](./05_ManageDataSources.md#assign-data-source).
+- You have defined the mapping settings for this definition, see [Manage mapping settings](./06_ManageMappingSettings.md).
+- You have prepared the related workflow to send the messages when they are needed in the process, see [Prepare workflow](../Operation/07_PrepareWorkflow.md).
+
+#### Procedure
+
+*DataHub Exporter > Exports*
+
+![Exporter definitions](../../Assets/Screenshots/EDI/Operation/ExportDefinitions.png "[Exporter definitions]")
+
+1. Click the definition you want to activate.   
+    The *Edit definition "definition name"* view is displayed. The *Settings* tab is displayed by default.
+
+    ![Edit definition](../../Assets/Screenshots/EDI/Operation/ExportDefinitionEdit.png "[Edit definition]")
+
+2. Click the *Active* toggle.   
+    The toggle is highlighted.
+
```

Comment:
Voschlag: Enable the *Active* toggle to activate the selected definition. By default, this toggle is disabled.

(So ist es meinstens in der bestehenden Doku beschrieben. Stimmt das immer, dass es "by default, disabled" ist?).

Reply:
ja stimmt